cloud-functions/src/utils/fetchTokenUSDPrice.ts

```
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+interface CoinGeckoResponse {
+	[coinId: string]: { usd: number; usd_24h_change?: number };
+}
+
+function formatUSDWithUnits(usd: String, numberAfterDot?: number) {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)